graviti/schema/builtin.py

```diff
@@ -337,3 +420,6 @@ class tensor(PortexBuiltinType):  # pylint: disable=invalid-name
     def __init__(self, shape: Iterable[Optional[int]], dtype: str, nullable: bool = False) -> None:
         super().__init__(shape=shape, dtype=dtype, nullable=nullable)
         self.shape = tuple(shape)
+
+    def _pyarrow_type_factory(self) -> "paDataType":
+        return pa.list_(_TYPE_TO_PYARROW[self.dtype])
```

Comment:
for tensor type, the dtype is portex type name, normally we need to search the portex imports to find the corresponding portex type, then transfer it to a pyarrow type.
Here, because the dtype can only be builtin numeric type, so your code works. What if we can put a Box2D in the dtype?

Reply:
why tensor type will be transferred to `pa.list_`?